src/libpgmoneta/utils.c

```diff
@@ -898,7 +898,27 @@ pgmoneta_set_proc_title(int argc, char** argv, char* s1, char* s2)
 
    // keep track of how long the title is now
    max_process_title_size = size;
+#elif defined(HAVE_OSX)
+   struct configuration* config;
```

Comment:
Move to top of function

Reply:
Should I move the declaration to the top of the function, or should I replace the existing condition with `HAVE_LINUX`?

Reply:
Move the common structures outside of the `#ifdef`'s

Reply:
However, there is also an `else` condition in which these elements are not utilized.

Reply:
I think in this case it would be ok